### PR TITLE
user/pulsemixer: new package

### PR DESCRIPTION
**Pull request by denisstrizhkin**
could maybe go to contrib

### CODE REVIEW

**Comment by q66** [comment]
this does not seem to use hatch? uses setuptools

**Comment by denisstrizhkin** [comment]
fails if i remove it

**Comment by nekopsykose** [comment]
because hatch-vcs pulls in setuptools

it's still wrong and you just need setuptools

**Comment by q66** [comment]
doesn't fail if you actually add the buildsystem it uses, which is setuptools

File: user/pulsemixer/template.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pkgver = "1.5.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = ["python-build", "python-installer", "python-setuptools"]
+depends = ["libpulse"]
+pkgdesc = "CLI and curses mixer for PulseAudio"
+maintainer = "Denis Strizhkin <strdenis02@gmail.com>"
+license = "MIT"
+url = "https://github.com/GeorgeFilipkin/pulsemixer"
+source = f"https://github.com/GeorgeFilipkin/{pkgname}/archive/{pkgver}.tar.gz"
```
Comment (nekopsykose) [inline user/pulsemixer/template.py:11]:
```suggestion
source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
```

**Pull request merged**